cedar-policy/CHANGELOG.md

```diff
@@ -73,6 +77,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fixed Wasm build script to be multi-target in JS ecosystem (#933)
 
 ## [3.2.0] - 2024-05-17
+Cedar Language Version: 3.2.0
```

Comment:
I think #766 might qualify as a language minor, making this 3.3.0?

Reply:
Ah, I agree.